src/P/List.hs

```diff
@@ -18,7 +19,8 @@ import qualified Data.Set as Set
 -- | /O(n log n)/ Remove duplicate elements from a list.
 --
 --   Unlike 'Data.List.nub', this version requires 'Ord' and runs in
---   /O(n log n)/ instead of /O(n²)/.
+--   /O(n log n)/ instead of /O(n²)/. Like 'Data.List.nub' the output may not
+--   be sorted. See 'sortNub' for sorted behaviour.
```

Comment:
Maybe would change the wording slightly:
`Like 'Data.List.nub' the output order is identical to the input order`